src/Cdn/Cdn.Autorest/docs/Invoke-AzCdnCommitProfileToAFDMigration.md

```diff
@@ -0,0 +1,190 @@
+---
+external help file:
+Module Name: Az.Cdn
+online version: https://learn.microsoft.com/powershell/module/az.cdn/invoke-azcdncommitprofiletoafdmigration
+schema: 2.0.0
+---
+
+# Invoke-AzCdnCommitProfileToAFDMigration
+
+## SYNOPSIS
+Commit the migrated Azure Front Door(Standard/Premium) profile..
+
+## SYNTAX
+
+```
+Invoke-AzCdnCommitProfileToAFDMigration -ProfileName <String> -ResourceGroupName <String>
+ [-SubscriptionId <String>] [-DefaultProfile <PSObject>] [-AsJob] [-NoWait] [-PassThru] [-Confirm] [-WhatIf]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Commit the migrated Azure Front Door(Standard/Premium) profile..
```

Comment:
Could you please provide more details regarding what this step means? Additionally, what are the consequences if a customer conducts this step?